### DISCUSSION TITLE
Disable preset change for program change messages

### DISCUSSION THREAD

**Discussion by nicolasvair** [shanteacontrols/OpenDeck#281] [Q&A] [answered]
Hi, 

I'm using the open deck M board with its midi output in my setup, using midi thru (usb in to DIN out). But when i send a program change message for my synth, the preset of open deck change.. 

Would it be possible to disable this way of working ? Or specify a channel ? Right now it seems program change on all the channels  trigger preset change in opendeck board...

Thank you !! I love this board. I had to find some workarounds at first but its very very stable, I love it.

**Comment by paradajz** [comment, marked as answer]
I'm not sure why I haven't made that optional, but the option is now implemented and is disabled by default. You'll see the new option in the configurator.  The firmware isn't released yet though.

**Comment by nicolasvair** [comment]
Hvala lepo, you’re the best ! 

Any idea when you’d release the firmware ?

**Comment by paradajz** [comment]
Released
https://github.com/shanteacontrols/OpenDeck/releases/tag/v7.8.0
